.github/workflows/cli_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+set -e -u -o pipefail
+
+VERSION=${1:-stable}
+
+check_file () {
+    if [[ ! -f "$1" ]] ; then
+        echo "Failed to create out.cil"
```

Comment:
This should probably say "Failed to create $1", right?  Since we call it with "-o new.cil" once.